admin/src/main/resources/io/buoyant/admin/css/dashboard-shared.css

```diff
@@ -36,3 +36,22 @@
 .failure-metric {
   color: #D84A4A;
 }
+
+/*
```

Comment:
These are from `summary.css`, which I believe we can delete once we delete the old dashboard.